### PR TITLE
targets: Use the meta refresh handler for offline update

### PR DESCRIPTION
**Pull request by mike-sul**
Use the new ota-lite handler that refreshes timestamp and snapshot metadata for Offline Updates. It allows a user to get TUF metadata for offline update that have a desired validity period, limited to 180 days.

Signed-off-by: Mike <mike.sul@foundries.io>

### CODE REVIEW

**Comment by mike-sul** [comment]
This PR should be merged after the ota-lite part https://git.foundries.io/development/cloudplatforms/infrastructure/ota-lite/-/merge_requests/325 is merged and deployed.

**Comment by mike-sul** [comment]
@doanac I tested it against https://git.foundries.io/development/cloudplatforms/infrastructure/ota-lite/-/merge_requests/325

File: subcommands/targets/offline-update.go [hunk 1]
```diff
@@ -179,11 +182,24 @@ func downloadTufRepo(factory string, tag string, prod bool, dstDir string) error
...
+	if err != nil {
+		return err
+	}
+	metadataNames := []string{
+		"timestamp", "snapshot", "targets",
 	}
-	for _, file := range metadataFileNames {
-		err = downloadMetadataFile(file)
+	for _, metaName := range metadataNames {
+		f, err := os.Create(path.Join(dstDir, metaName+".json"))
```
Comment (doanac) [inline subcommands/targets/offline-update.go:193]: I think [os.WriteFile](https://pkg.go.dev/os#WriteFile) would save you some a few lines of code and error handling.

Comment (mike-sul) [reply subcommands/targets/offline-update.go]: done, forced push.

**Comment by mike-sul** [comment]
Added the `tuf-only` flag https://github.com/foundriesio/fioctl/pull/187/commits/aa3c0b248fb2cc7cce152c2d334dfb17582e08ea

**Comment by mike-sul** [comment]
I realized that we need to download root starting from version #3 https://github.com/foundriesio/fioctl/pull/187/commits/aa02d77c19ceb9a7e52824b71971d83c7ae0e91a.

**Review by doanac — COMMENTED** [review]
looks good - we can just drop the TODO comment.

**Comment by mike-sul** [comment]
> looks good - we can just drop the TODO comment.

done

**Review by doanac — APPROVED** [review]

**Pull request merged**